Prose/ProseLib/Sources/SidebarFeature/Footer/Footer.swift

```diff
@@ -58,40 +58,43 @@ struct Footer: View {
 
 // MARK: Reducer
 
-public let footerReducer: Reducer<
-    (FooterState, UserCredentials),
+let footerReducer: Reducer<
+    FooterState,
     FooterAction,
     Void
 > = Reducer.combine([
```

Comment:
One day I should rewrite all `: Reducer<...> = Reducer.combine([` to `= Reducer<...>.combine([`, but that's for another time 😌